script/check-broken-links.js

```diff
@@ -26,6 +28,17 @@ function checkBrokenLinks() {
 
       const isApp = !!entryname;
       if (isApp) ignorePaths.push(path);
+
+      // if we have lazy images, they don't have a src attr,
+      // so we need to add one for the blc
+      if (file.contents.includes('data-src=')) {
+        lazyImageFiles.push(fileName);
+        const doc = cheerio.load(file.contents);
+        doc('img[data-src]').each((i, el) => {
+          doc(el).attr('src', doc(el).attr('data-src'));
+        });
+        file.contents = doc.html();
+      }
```

Comment:
Could we possibly chunk this and the cleanup logic into their own function above `checkBrokenLinks`? Just worried that the comment could get lost or something.